rbi/stdlib/csv.rbi

```diff
@@ -277,11 +278,12 @@ class CSV < Object
   sig do
     params(
         path: T.any(String, ::Sorbet::Private::Static::IOLike),
+        mode: String,
         options: T::Hash[Symbol, BasicObject],
```

Comment:
if options is declared as a kwsplat now, both this type and the type on 273 should say `options: BasicObject` instead of `T::Hash[Symbol, BasicObject]`.

Reply:
Fixed, and thanks for helping out. First time opening a PR so the extra help is totally appreciated.